src/components/Aside/Aside.tsx

```diff
@@ -0,0 +1,108 @@
+import { useEffect, useRef } from "react";
+import styles from "./Aside.module.css";
+import Seed from "../../assets/Seed.png";
+import Sprout from "../../assets/Sprout.png";
+import YoungTree from "../../assets/YoungTree.png";
+import LargeTree from "../../assets/LargeTree.png";
+import Github from "../../assets/Github.png";
+import { Grade } from "../../api/services/common/types.ts";
+
+interface AsideProps {
+  githubUsername: string;
+  easyProgress: string;
+  mediumProgress: string;
+  hardProgress: string;
+  solvedTasks: number;
+  totalTasks: number;
+  profile_url: string;
```

Comment:
prop 이름은 camelCase로 일관되게 맞춰주셨으면 좋겠어요.

```suggestion
  profileUrl: string;
```

이미 암묵적으로 따르는 관행인 것 같지만 그래도 관련 내용 기여 가이드에 추가하였습니다.

https://github.com/DaleStudy/leaderboard/wiki/%EA%B8%B0%EC%97%AC-%EA%B0%80%EC%9D%B4%EB%93%9C#%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-%EA%B5%AC%EC%A1%B0